sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -40,11 +40,13 @@ static StorageTestBase()
         private const string CopySourceAuthorization = "x-ms-copy-source-authorization";
         private const string PreviousSnapshotUrl = "x-ms-previous-snapshot-url";
         private const string FileRenameSource = "x-ms-file-rename-source";
+        private const string SasVersion = "sv";
 
         public StorageTestBase(bool async, RecordedTestMode? mode = null)
             : base(async, mode)
         {
             SanitizedQueryParameters.Add(SignatureQueryName);
+            IgnoredQueryParameters.Add(SasVersion);
```

Comment:
`HeaderRegexSanitizers.Add(new HeaderRegexSanitizer("x-ms-copy-source", value: "sanitized-value", regex: "(?:[?&](sv)=)(?<secret>[^&\\\"\\s\\n,\\\\]+)", groupForReplace: "secret"))`

Reply:
@scbedd are you sure this regex is correct?  I attempted to add this in my PR, but it appears to be ineffective.

Reply:
I'm pretty confident yeah. It's a slight modification on one of the [default sanitizer](https://github.com/Azure/azure-sdk-tools/blob/470c31d82ff1cd8d36bfb42d9edbd62d497fe8f9/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs#L103_ that applies for all SAS urls.

Best way to repro this is just to kick a job on playback yea? Let me try this locally.

Reply:
@scbedd I pushed the change to this PR, if you run the tests locally in playback mode, you will see the regex appears to be ineffective.

Reply:
Definitely see that in the CI run. I should be able to get this workin.

Reply:
Ok I am 99% certain I see the issue. I pushed a new commit. Should resolve it.

Reply:
Yep checking out across the board.